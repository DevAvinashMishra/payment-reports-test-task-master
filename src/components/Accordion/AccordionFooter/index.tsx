import React from "react";
import Typography from "@mui/material/Typography";

type AFProps = {
  value: string;
};

const AccordionFooter = ({ value = "" }: AFProps) => {
  return (
    <>
      <Typography
        sx={{
          fontWeight: 700,
          marginTop: "1rem",
          backgroundColor: "aliceblue",
          padding: "1rem",
          borderRadius: "5px",
          width: "100%",
        }}
      >
        {value}
      </Typography>
    </>
  );
};

export default AccordionFooter;
