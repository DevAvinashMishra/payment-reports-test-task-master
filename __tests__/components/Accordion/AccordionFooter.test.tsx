/* eslint-disable no-console */
import React from "react";
import { shallow } from "enzyme";
import AccordionFooter from "../../../src/components/Accordion/AccordionFooter";

describe("<AccordionFooter />", () => {
  const tree = shallow(<AccordionFooter value="1234" />);

  describe("Should Render", () => {
    it("Should match snapshot", () => {      
      expect(tree).toMatchSnapshot();
    });
  });
});
