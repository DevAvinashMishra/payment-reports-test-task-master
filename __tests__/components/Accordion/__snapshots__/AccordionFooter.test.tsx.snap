// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccordionFooter /> Should Render Should match snapshot 1`] = `ShallowWrapper {}`;
